docs/demo-video-instruction.md

```
@@ -0,0 +1,206 @@
+* Adding the framework wrapper
+* Rendering the share button in a component or page
+
+---
+
+## 3. Show the Integration Code
+
+Briefly walk through the relevant integration files added in your PR.
+
+Examples:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)